docs/root/configuration/overview/mgmt_server.rst

```diff
@@ -53,6 +53,7 @@ The following statistics are generated for all subscriptions.
  :widths: 1, 1, 2
 
  config_reload, Counter, Total API fetches that resulted in a config reload due to a different config
+ config_reload_time, Gauge, Timestamp of the last config reload as milliseconds since the epoch
```

Comment:
config_reload is only applicable/implemented for RDS/SRDS and VHDS - but added here generically. I followed the same pattern and added this new stat here even though it is implemented only for RDS and SRDS. If it is better to move to rds stats page - I can move.